tests/test_fe_handlers.py

```diff
@@ -100,3 +100,72 @@ def test_usertokens(session, users, http_client, base_url):
             headers={'X-Grouper-User': user.username})
     assert resp.code == 200
     assert "Disabled token: myFoobarToken" in resp.body
+
+@pytest.mark.gen_test
+def test_request_emails(graph, groups, permissions, session, standard_graph, users, base_url,
+                        http_client):
+    tech = groups["tech-ops"]
```

Comment:
thanks for doing this!

any reason you decided to do an entirely new test rather than adding email count checks to `test.test_requests.test_basic_request`?